### PR TITLE
Fixed #203 -- Added a back to top button in documentation

### LINKED ISSUE #203: Back to top button
This is more of a usability request than an issue.  I'd love to see a back to top button - particularly on the documentation where some of the pages are really long.  I have always imagined this to be next to the version selector, but there may be a better place.

Congrats on the redesign :)

### PR DESCRIPTION
**Pull request by patjouk**
I've added a button above the language and version switchers. See screenshot:

backtotop

### CODE REVIEW

File: djangoproject/templates/docs/doc.html [hunk 1]
```diff
@@ -99,8 +105,9 @@
          <strong>{% if release.is_dev %}development{% else %}{{ version }}{% endif %}</strong>
        </span>
     </li>
-  </ul>
-</div>
+  </ul> 
+
+</div>
```
Comment (charettes) [inline djangoproject/templates/docs/doc.html]: Please revert this changes.

File: djangoproject/scss/_style.scss [hunk 2]
```diff
@@ -2230,6 +2230,15 @@ h2 + .list-link-soup {
...
             }
+			&.justone{
+            	padding: 0;
+            	border: 0;
+            	border-radius: 0;
+			  &::before {
+			  	@include fa-icon();
+			  	content: $fa-var-chevron-up;
+			  }
+			}
```
Comment (charettes) [inline djangoproject/scss/_style.scss]: It looks like there's a mix of tabs and spaces here. Please only use spaces.

**Comment by patjouk** [comment]
Indents and weird edit of lines 108 to 110 have been fixed.

File: djangoproject/templates/docs/doc.html [hunk 3]
```diff
@@ -54,6 +54,13 @@
 
 {% block content %}
 <div id="version-switcher">
+
+  <ul id="totop">
+    <li class="current">
+      <a class="justone" href="#top" title="Back to top"></a>
```
Comment (timgraham) [inline djangoproject/templates/docs/doc.html]: Is the justone class doing something? I didn't see a reference to it in our own CSS.

File: djangoproject/scss/_style.scss [hunk 4]
```diff
@@ -2197,7 +2197,7 @@ h2 + .list-link-soup {
     }
 }
 
-#doc-versions, #doc-languages {
+#doc-versions, #doc-languages, #backtotop-switchers {
```
Comment (timgraham) [inline djangoproject/scss/_style.scss]: `#backtotop-switcher` (chop 's')

File: djangoproject/scss/_style.scss [hunk 5]
```diff
@@ -2230,6 +2230,15 @@ h2 + .list-link-soup {
                 color: $green;
                 border: 1px solid $green-light;
             }
+            &.backtotop-switchers {
+                padding: 0;
+                border: 0;
+                border-radius: 0;
```
Comment (timgraham) [inline djangoproject/scss/_style.scss]: unneeded

**Comment by timgraham** [comment]
As we discussed, it could be nice if the entire button were clickable rather than only the center part where the chevron appears.

**Comment by patjouk** [comment]
Hi @timgraham. Thanks again for your help. I'll try to work on this PR this week.

File: djangoproject/scss/_style.scss (hunk 4 above)
Comment (tBaxter) [inline djangoproject/scss/_style.scss]: stylistically, wouldn't #back-to-top-switcher be more correct?  I also question "switcher" as nothing is being switched.

**Comment by timgraham** [comment]
Closing due to inactivity.
